cpp/libsarmata-client/sarmata_client.h

```diff
@@ -0,0 +1,39 @@
+#ifndef __SARMATA_CLIENT_H__
+#define __SARMATA_CLIENT_H__
+
+#include "sarmata_asr.grpc.pb.h"
+
+
+namespace techmo { namespace sarmata {
+
+struct SarmataClientConfig {
```

Comment:
Could we change the name of this struct to something more related to ASR? This is not a config of SarmataClient, maybe naming it `RequestConfig` ? or `ASRSessionConfig`? would be more appropriate?

Reply:
`SarmataSessionConfig` would be appropriate. `ASR-` could be confused with the other ASR product. Also, it is used in the server's code. And `RequestConfig` is already in a request.